src/library/basetracktablemodel.cpp

```diff
@@ -1105,3 +1116,15 @@ bool BaseTrackTableModel::updateTrackMood(
     return m_pTrackCollectionManager->updateTrackMood(pTrack, mood);
 }
 #endif // __EXTRA_METADATA__
+
+void BaseTrackTableModel::slotCoverFound(
+        const QObject* pRequester,
+        const CoverInfo& coverInfo,
+        const QPixmap& pixmap) {
+    Q_UNUSED(pixmap);
+    if (pRequester != this ||
+            getTrackLocation(m_toolTipIndex) != coverInfo.trackLocation) {
+        return;
+    }
+    emit dataChanged(m_toolTipIndex, m_toolTipIndex, {Qt::ToolTipRole});
```

Comment:
What has the tooltip to do with the cover?

Reply:
If library tooltips are enabled and you move the mouse over a library cover, than the tooltip shows a bigger picture of this cover (mouse pointer is not captured by screenshot-tool):
![grafik](https://github.com/mixxxdj/mixxx/assets/64457745/d745dd46-584b-4738-b200-7cb5c51bb904)

Reply:
Got it!t Thanks! Still discovering mixxx features I never use :-)